packages/fiber/src/core/events.ts

```diff
@@ -170,12 +167,19 @@ export function createEvents(store: UseStore<RootState>) {
       const localState = { stopped: false }
 
       for (const hit of intersections) {
-        const hasPointerCapture = (id: number) => internal.capturedMap.get(id)?.eventObject === hit.eventObject
+        const hasPointerCapture = (id: number) => internal.capturedMap.get(id)?.has(hit.eventObject)
```

Comment:
This function will return true or false or undefined, which seems confusing. Perhaps add a `?? false` to the end?

Reply:
making the change in a patch